styles.css

```diff
@@ -0,0 +1,102 @@
+/* Adding poppins fonts */
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+* {
+  font-family: Poppins, sans-serif;
+}
+
+/* adding the main container */
+
+.container {
+  min-width: 375px;
+  max-width: 1200px;
+  padding-bottom: 70px;
+  background-color: #fff;
+  background-repeat: no-repeat;
+  border-bottom-left-radius: 80px;
+  position: relative;
+  background-image: url(images/header-shapes-mobile.jpg);
```

Comment:
- [ ] I can't see any background image. The images you have used here are not `displaying` as there is no `image` folder in your GitHub repository. Kindly fix this issue.